ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -267,7 +267,7 @@ message GrantUserRightsRequest {
     string user_id = 1;
 
     // The rights to grant.
-    // Required
+    // Optional
```

Comment:
Looks like we've always allowed to grant an empty list of rights and so this field is really an optional one. Reflecting that in the proto comment